docs/build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```

Comment:
Please remove the `docs/build` folder, this one should be flagged by the gitignore and not included in the repository.

Reply:
Done